api/src/main/java/org/apache/iceberg/util/CharSequenceSet.java

```diff
@@ -166,6 +166,7 @@ public void clear() {
     wrapperSet.clear();
   }
 
+  @SuppressWarnings("CollectionUndefinedEquality")
```

Comment:
`containsAll` uses `CharSequenceWrapper` for equals to ensure consistent
 hashing and equals behaviour similar to `CharSequenceMap` (javadoc of `CharSequenceMap` explains it). But the tool is unable to figure it out I guess. Hence, suppressing here and below two places.